### PR TITLE
vmui: correctly display range results in Table view

### PR DESCRIPTION
**Pull request by Loori-R**
- #3516

### CODE REVIEW

File: app/vmui/packages/vmui/src/components/Views/TableView/TableView.tsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ const TableView: FC<GraphViewProps> = ({ data, displayColumns }) => {
         : (d.metric[c.key] || "-")
       )),
       value: d.value ? d.value[1] : "-",
+      values: d.values ? d.values.map(([time, val]) => `${val} @${time}`) : null,
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/components/Views/TableView/TableView.tsx:51]: I think it is better to return an empty array instead of null

Comment (Loori-R) [reply app/vmui/packages/vmui/src/components/Views/TableView/TableView.tsx]: updated

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@Loori-R , thanks for the fix!

**Pull request merged**